hack/k8s/helm/nuclio/README.md

```diff
@@ -63,37 +63,42 @@ helm install \
 ```
 
 ### Install on Minikube using a local, insecure registry
-By clearing `registry.secretName`, Nuclio will not try to load Docker secrets.
+
+You will need to run a local Docker registry. Run the following command on the host if you're working with Docker for Mac or on the Minikube VM:
+```sh
+docker run -d -p 5000:5000 registry:2
+```
+
+By not providing a registry secret name (`registry.secretName`) nor credentials (`registry.credentials.username` / `registry.credentials.password`), Nuclio will understand credentials are not needed, and not try to load Docker secrets.
 
 ``` sh
 helm install \
-	--set registry.secretName= \
+    --set registry.pushPullUrl=localhost:5000 \
```

Comment:
👏

Reply:
The indentation here is consistent with the rest of the doc (maybe github messes it up on display)